People/Doctor.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+
+namespace People
+{
+    public class Doctor
+    {
+        private const int PerPatientBonus = 100;
+        private const int QualifiedDoctorPayment = 15000;
+        private const int UnderQualifiedDoctorPayment = 1000;
+        private string _name;
+        private int _age;
+        private int _id;
+        private string _gender;
+        private string _specialty;
+        private int _yearsOfExperience;
+        private bool _hasPhd;
+        private int _salary;
+        private string[] _patientsArray;
+
+        public Doctor(string name, string gender, int age, int id, string specialty, int yearsOfExperience, bool hasPhd)
+        {
+            _name = name;
+            _gender = gender.ToLower();
+            _age = age;
+            _id = id;
+            _specialty = specialty.ToLower();
+            _yearsOfExperience = yearsOfExperience;
+            _hasPhd = hasPhd;
+        }
+
+        public Doctor(string name, string gender, int age, int id, string specialty, int yearsOfExperience, bool hasPhd, string[] patients)
+        {
+            _name = name;
+            _gender = gender.ToLower();
+            _age = age;
+            _id = id;
+            _specialty = specialty.ToLower();
+            _yearsOfExperience = yearsOfExperience;
+            _hasPhd = hasPhd;
+            _patientsArray = patients;
+        }
+
+        public int GetSalary()
+        {
+            if (_hasPhd && _yearsOfExperience > 5 || _specialty == "surgeon" || _specialty == "dentist")
+            {
+                _salary = QualifiedDoctorPayment + (_patientsArray.Length * PerPatientBonus);
+            }
+            else
+            {
+                _salary = UnderQualifiedDoctorPayment * _yearsOfExperience + (_patientsArray.Length * PerPatientBonus);
+            }
+
+            return _salary;
+        }
+
+        public void OutPutDoctorInfo()
+        {
+            Console.WriteLine("Doctor's Information:");
+            Console.WriteLine($"{_id}, {_name}, {_age} years old, Salary: {GetSalary()}, {_gender}, Has PHD: {_hasPhd}, \nYears Of Experience: {_yearsOfExperience}, Specialization: {_specialty}");
+            Console.WriteLine($"List of Patients: {String.Join(',', _patientsArray)}");
+            Console.WriteLine("------------------");
+        }
+
+        public void OutPutDoctorPatients()
+        {
+            Console.WriteLine($"List of Patients of Dr.{_name}: {String.Join(',', _patientsArray)}");
+            Console.WriteLine("------------------");
+        }
+
+        public string GetDoctorName() { return _name; }
+
+        public string[] GetListOfPatients() { return _patientsArray; }
+
+        public void AddPatientsToDoctorProfile(string[] patients)
+        {
+            _patientsArray = patients;
```

Comment:
In case the doctor already has some patients this method will override the data with the patients.
You can fix this problem in next home task